packages/pf4-component-mapper/src/components/form-template.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import formTemplate from '../../../common/src/form-template';
+
+import { Text, TextVariants } from '@patternfly/react-core/dist/js/components/Text/Text';
```

Comment:
Just leaving a note. Since we now build cjs/esm/umd builds we have to adapt these paths as well. I am afraid that the only possible sulution will be babel transform imports plugin.

Reply:
ajta krajta :snake:

Reply:
But with these paths already transformed, we can just change `js` so it should be pretty easy. No mappers. :smile_cat:

Reply:
Yup we might just replace the `/js/` with other equivalents